Dockerfile

```diff
@@ -8,4 +8,6 @@ RUN go install ./cmd/zed
 FROM alpine:3.13
 RUN apk --no-cache add ca-certificates
 COPY --from=build /go/bin/* /usr/local/bin/
+FROM gcr.io/distroless/base
+COPY zed /
```

Comment:
Let's replace the existing alpine with distroless, just bring over the copy command.